.github/workflows/js-tests.yml

```diff
@@ -1,6 +1,15 @@
 name: JS tests
 
-on: pull_request
+on:
+  pull_request:
+    paths-ignore:
+    - '**.php'
+    - '**.md'
+  push:
+    branches: [master]
+    paths-ignore:
+    - '**.md'
+    - '**.php'
```

Comment:
I believe this will break on a PR that only touches *.md and *.php files. The workflow won't run, so the required checks won't ever be set, so it'll sit there waiting forever for them to show up.

https://github.community/t/feature-request-conditional-required-checks/16761

That's why php-linting.yml has a "detect changed files" step and then uses `job.<id>.if` to skip jobs instead of using `paths`.

You can test it by creating a PR that wants to merge into this PR, and seeing if those checks ever show up.

Also, we probably do still want the DangerCI check to run even if only .php or .md are changing, since it's for checking the PR comment rather than the code. I'm already working on #18001 to split that out for other reasons.

Reply:
oh, right. I guess I'll just remove `paths-ignore` altogether